Dockerfile

```diff
@@ -36,6 +36,11 @@ RUN rm irods-icommands.deb
 # Install test coverage module
 RUN pip install coverage
 
+# Install haystack and sphinx
+RUN pip install django-haystack
+RUN pip install sphinx

```

Comment:
We are not using sphinx, we are using solr. They are different search engines.

Also, this package is another thing entirely, the Sphinx Documentation Generator used by most well documented python projects